advocacy_docs/edb-postgres-ai/analytics/how_to_lakehouse_sync.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Lakehouse Sync
+navTitle: Lakehouse Sync
+description: How-to guide for performing a Lakehouse Sync.
+---
+
+## Overview
+
+A Lakehouse Sync is a way to capture information from a transactional database at a point in time and sync that information to a Managed Store Location in an AWS region.
+
+Once synced, the data in the MSL is organized into Lakehouse tables. These Lakehouse tables can then be queried by a Lakehouse node optimized for higher-performance queries using a vectorized query engine for Lakehouse tables.
+
+## Perfroming a Lakehouse Sync
+
+### Prerequisites
+
+- a Postgres cluster running on your EDB Postgres AI Cloud Service account
+
+### Navigate to Lakehouse Sync
+
+1. Login to your EDB Postgres AI Console.
+
+2. From the landing page, select the project with the database instance you want to sync. If it is not shown on the landing page, select the **View Projects** link in the **Projects** section and select your project from there.
+
+3. Select the **Migrate** dropdown in the left navigation bar and then select **Migrations**.
+
+4. Select the **Create New Migration** button.
+
+### Define Lakehouse Sync
+
+5. Give the sync a **Name**, then select a **Source Cluster** and the **Database** you want to sync.
+
+6. If you have already created a MSL you want to use, select that MSL from the list of available MSLs and skip to step 10. If not, select the **Create New Managed Storage Location** button to bring up the **Add Managed Storage Location** modal.
```

Comment:
Not sure if these are the correct terms, but I would change the wording a bit so non-native speakers can easily follow as well. 
```suggestion
6. If you have already created a MSL you want to use, select that MSL from the list of available MSLs and skip to step 10. If not, select the **Create New Managed Storage Location** button to open the **Add Managed Storage Location** pop-up window.
```